cmake/modules/protobuf.cmake

```diff
@@ -26,11 +25,11 @@ else()
 		message(STATUS "Using bundled protobuf in '${PROTOBUF_SRC}'")
 		ExternalProject_Add(protobuf
 			PREFIX "${PROJECT_BINARY_DIR}/protobuf-prefix"
-			DEPENDS openssl zlib
+			DEPENDS zlib
```

Comment:
Protobuf does not seem to need openssl.

Reply:
https://github.com/protocolbuffers/protobuf/blob/main/src/README.md#c-protobuf---unix